app/client/components/transferAccount/StandardTransferAccountList.tsx

```diff
@@ -182,7 +196,8 @@ class StandardTransferAccountList extends React.Component<
   updateQueryData(query: Query) {
     this.setState({
       params: query.params,
-      searchString: query.searchString
+      searchString: query.searchString,
+      page: 1
     });
   }
```

Comment:
Whenever there is a new query, we reset to page 1
https://ux.stackexchange.com/questions/129073/should-the-pagination-be-reset-when-changing-the-order